### PR TITLE
56 omnibridge first pass

### PR DESCRIPTION
**Pull request by CJBangert**
## What

- add details on how omnibridge works
- add contract addresses
 - added other general details

## Refs

- #56

### CODE REVIEW

File: docs/bridges/tokenbridge/omnibridge.md [hunk 1]
```diff
@@ -27,13 +27,13 @@ References:
...
-| --------------------- | ----------------------------------------------- |
-| Frontend URL          | https://omni.gnosischain.com                  |
+|-----------------------|-------------------------------------------------|
+| Frontend URL          | https://omni.gnosischain.com                    |
 | Trust Model           | [4-of-6 Validator Multisig](#bridge-validators) |
 | Governance            | [7-of-16 Multisig](#bridge-governance)          |
 | Governance Parameters | Validator Set, Daily Limits, Fees               |
-| Bug Bounty            | TODO                                            |
-| Bug Reporting         | TODO                                            |
+| Bug Bounty            | [upp to $2m](https://immunefi.com/bounty/gnosischain/) |
```
Comment (dan-gnosis) [inline docs/bridges/tokenbridge/omnibridge.md:35]: Typo - 'Up', not 'Upp'

File: docs/bridges/tokenbridge/omnibridge.md [hunk 2]
```diff
@@ -27,13 +27,13 @@ References:
...
+| Bug Reporting         | On the Immunefi site [here](https://immunefi.com/bounty/gnosischain/) |
```
Comment (dan-gnosis) [inline docs/bridges/tokenbridge/omnibridge.md:36]: Just say [Immunefi](https://immunefi.com/bounty/gnosischain/)

File: docs/bridges/tokenbridge/omnibridge.md [hunk 3]
```diff
@@ -43,23 +43,24 @@ References:
 
 #### Ethereum
 
-| Contract                      | Ethereum Address |
-| ----------------------------- | ---------------- |
-| Proxy Contract                |                  |
-| Validator Management Contract |                  |
+| Contract                        | Ethereum Address                           |
+|---------------------------------|--------------------------------------------|
+| Omnibridge Multi-Token Mediator | 0x88ad09518695c6c3712AC10a214bE5109a655671 |
```
Comment (dan-gnosis) [inline docs/bridges/tokenbridge/omnibridge.md:48]: Please link these addresses with the Etherscan links

File: docs/bridges/tokenbridge/omnibridge.md [hunk 4]
```diff
@@ -43,23 +43,24 @@ References:
...
 #### Gnosis
 
-| Contract                      | Gnosis Address |
-| ----------------------------- | -------------- |
-| Proxy Contract                |                |
-| Block Reward Contract         |                |
-| Validator Management Contract |                |
+| Contract                      | Gnosis Address                             |
+|-------------------------------|--------------------------------------------|
+| Proxy Contract                | 0x75Df5AF045d91108662D8080fD1FEFAd6aA0bb59 |
```
Comment (dan-gnosis) [inline docs/bridges/tokenbridge/omnibridge.md:56]: Please link the Blockscout or Gnosisscan addresses

File: docs/bridges/tokenbridge/omnibridge.md [hunk 5]
```diff
@@ -111,10 +112,25 @@ References:
 ## How it works
 
 ![](/img/bridges/diagrams/token-bridge.svg)
```
Comment (dan-gnosis) [inline docs/bridges/tokenbridge/omnibridge.md:114]: I'll help to clean this up tomorrow

**Review by dan-gnosis — CHANGES_REQUESTED** [review]
Good work. Some minor typos and improvements needed.

**Comment by dan-gnosis** [comment]
Looks good to go!

File: docs/bridges/tokenbridge/omnibridge.md (hunk 5 above)
Comment (alebanzas) [reply docs/bridges/tokenbridge/omnibridge.md:135]: Is this ready?

**Comment by alebanzas** [comment]
Thanks @CJBangert and @dan-gnosis .

There is a conflict with `main` to merge.

**Pull request merged**